test/smoke/testApps/Sampling/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'war'
 
 dependencies {
-    implementation aiWebJar
+    implementation aiCoreJar
```

Comment:
Is this a special case ? Still have core dependency?

Reply:
this test uses the 2.x SDK to send telemetry, to verify that telemetry from 2.x SDK is also sampled correctly